platform-hub-api/app/services/project_bills_query_service.rb

```diff
@@ -0,0 +1,24 @@
+module ProjectBillsQueryService
+  extend self
+
+  def fetch project_id
+    # Don't allow } and ' as they can be used for SQL injection in this particular case
+    raise ArgumentError if project_id.count("}'") > 0
```

Comment:
Could potentially sanitize query below to automatically take care of it?

Reply:
Can't use regular SQL quote sanitisation as the below query is a special Postgres syntax (sanitisation breaks it).

Reply:
Ah! all good then.